src/promptflow/promptflow/_cli/_pf/_tool.py

```diff
@@ -151,3 +183,27 @@ def list_tool(args):
     pf_client = PFClient()
     package_tools = pf_client._tools.list(args.flow)
     print(json.dumps(package_tools, indent=4))
+
+
+@exception_handler("Tool validate")
+def validate_tool(args):
+    import importlib
+
+    pf_client = PFClient()
+    try:
+        __import__(args.source)
```

Comment:
what if args.source is None ?

Reply:
mark --source as requried. If user provide a invalid source, cli will raise user error.
![image](https://github.com/microsoft/promptflow/assets/17938940/850b04ba-9b12-442a-8a52-02e06ccf9301)